wgpu-hal/src/dx12/mod.rs

```diff
@@ -541,11 +545,23 @@ pub struct Surface {
     target: SurfaceTarget,
     supports_allow_tearing: bool,
     swap_chain: RwLock<Option<SwapChain>>,
+    options: wgt::Dx12BackendOptions,
 }
 
 unsafe impl Send for Surface {}
 unsafe impl Sync for Surface {}
 
+impl Surface {
+    /// Returns the waitable handle of the swapchain, if it exists.
+    pub unsafe fn waitable_handle(&self) -> Option<Foundation::HANDLE> {
+        let swapchain = self.swap_chain.read();
+        if let Some(swap_chain) = swapchain.as_ref() {
+            return swap_chain.waitable.clone();
```

Comment:
This implies that Handles are refcounted, but they are Copy aliases around `*const c_void` - we should return it by copy

Reply:
so just drop the `clone()`?

Reply:
Yeah